Replace image for SP initiated login (#3208)
SumoLogic/sumologic-documentation: docs/manage/security/saml/integrate-sumo-with-azure-ad.md

    2. **X.509 Certificate**. Use a text editor to open the certificate file you downloaded in substep 12 of [Configure Sumo as an Enterprise App in Azure AD](#configure-sumo-as-an-enterprise-app-in-azure-ad). Copy and paste the contents of the file into the field.   
1. **Attribute Mapping.** Select **Use SAML subject.**
1. **Configure SP-initiated Login.** (Optional) This step has instructions for setting up SP-initiated login. When SP-initiated login has been enabled, your SAML configuration will appear as an additional authentication option within your subdomain-enabled account login page. SP initiated login requires a custom Sumo Logic subdomain. If a custom subdomain has not yet been configured for your org, following the instructions in the [Change account subdomain](/docs/manage/manage-subscription/manage-org-settings#change-account-subdomain) section of the **Manage Organization** topic.

>    1. Click **SP Initiated Login Configuration** in the **Optional Settings** section of the SAML configuration page if not already selected. When you click this option, the following configurations appear. <br/><img src={useBaseUrl('/img/security/sp-init-settings.png')} alt="SP-initiated login" style={{border: '1px solid black'}} width="300" />

Reviewer: I'd reorder so that instructions are cascading

```suggestion
   1. Under the SAML configuration page > **Optional Settings** section, make sure that **SP Initiated Login Configuration** is selected. When you click this option, the following configurations appear. <br/><img src={useBaseUrl('/img/security/sp-init-settings.png')} alt="SP-initiated login" style={{border: '1px solid black'}} width="300" />
```
